nifi-commons/nifi-utils/src/main/java/org/apache/nifi/util/file/FileUtils.java

```diff
@@ -549,49 +548,22 @@ public static void syncWithRestore(final File primaryFile, final File restoreFil
      * @throws IOException if the MD5 hash could not be computed
```

Comment:
@exceptionfactory Suggestion to change the name to the hashing algorithm being used. Same for above description.
```suggestion
     * @throws IOException if the SHA-256 hash could not be computed
```

Reply:
Thanks for catching that @mtien-apache.  Will update the documentation.